wikichange/src/content/content.js

```diff
@@ -18,7 +18,13 @@ const insertAfter = (newNode, existingNode) => {
  */
 const title = (() => {
     let titleSpan = document.getElementsByClassName("mw-page-title-main");
-    let title = titleSpan[0].innerHTML;
+    let title;
+    if (titleSpan.length == 0) {
+        let url = document.URL.split("/");
+        title = url[url.length - 1];
```

Comment:
Can you please add a regex to replace '_' with a space? For a page like https://en.wikipedia.org/wiki/Death_Note, this will parse the title as `Death_Note` while a page like https://en.wikipedia.org/wiki/United_States will be parsed as `United States`. it is not showing any issue now, but since we are using title for most of our stuff, it should be better to keep the format similar in case we run into a problem because of this in the future.